packages/builder/src/pages/builder/auth/_layout.svelte

```diff
@@ -10,13 +10,18 @@
     $redirect("../")
   }
 
-  if ($admin?.checklist?.branding) {
+  if ($admin.cloud && $admin?.checklist?.branding) {
```

Comment:
NAB: Can `$admin` be null?

Reply:
Pretty sure no, but better to be safe than sorry.